pkg/catalog/catalog.go

```diff
@@ -1770,29 +1771,37 @@ type GCUncommittedMark struct {
 }
 
 type PrepareGCUncommittedInfo struct {
-	Mark     *GCUncommittedMark
-	Metadata graveler.GarbageCollectionRunMetadata
+	RunID      string `json:"run_id"`
+	Location   string `json:"location"`
+	ObjectName string `json:"object_name"`
+	Mark       *GCUncommittedMark
```

Comment:
I think these changes are unrelated to the PR and should be done in a different issue

Reply:
I've returned the ObjectName as part of the response.
So I've used different return object type.
The object type that was used was defined in proto file - this change I can move to a different PR.
But I wanted to remove the unused field.
So I also moved it out of the proto as it is not serialized as proto.